scripts/qa_crowbarsetup.sh

```diff
@@ -3392,9 +3447,8 @@ EOF
         test $cephret -eq 0 || ret=104
     fi
 
-    if [ "$wanttempest" = "1" ]; then
-        scp $novacontroller:/var/lib/openstack-tempest-test/tempest.log .
-        scp $novacontroller:/var/lib/openstack-tempest-test/tempest.subunit.log .
+    if [[ $wanttempest ]] ; then
+        scp $novacontroller:/var/lib/openstack-tempest-test/tempest*.log .
```

Comment:
No, the thing is that there is a codepath that do not create the subunit file.

IMHO the place that collect the tempest logs needs to assert a minimum set of file that needs to be there